.github/workflows/update_infra.yml

```diff
@@ -0,0 +1,63 @@
+name: Open a PR in Infra repository
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [ closed ]
+    paths:
+      - 'gpd/openapi/openapi.json'
+      - 'payments/openapi/openapi.json'
+      - 'reporting-analysis/openapi/openapi.json'
+
+
+  workflow_dispatch:
+
+jobs:
+  pull-request:
+    if: ${{ github.event.pull_request.merged }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # set repo
+      - name: Set GPD repo
+        run: |
+          echo "REPO=gpd" >> $GITHUB_ENV
+          echo "FOLDER=v1" >> $GITHUB_ENV
+        if: ${{ contains(github.event.pull_request.labels.*.name, 'gpd') }}
+
+      - name: Set Payments repo
+        run: |
+          echo "REPO=payments" >> $GITHUB_ENV
+          echo "FOLDER=payments/v1" >> $GITHUB_ENV
+        if: ${{ contains(github.event.pull_request.labels.*.name, 'payments') }}
+
+      - name: Set Reporting Analysis repo
+        run: |
+          echo "REPO=reporting-analysis" >> $GITHUB_ENV
+          echo "FOLDER=reporting-analysis/v1" >> $GITHUB_ENV
+        if: ${{ contains(github.event.pull_request.labels.*.name, 'reporting-analysis') }}
+
+
+      # prepare openapi template for infra repo
+      - run: |
+          mkdir -p "${GITHUB_WORKSPACE}/infra"
+          jq  '."servers"[0]."url" |= "${host}"' "${GITHUB_WORKSPACE}/${{env.REPO}}/openapi/openapi.json" > "${GITHUB_WORKSPACE}/infra/_openapi.json.tpl"
+
+      # open a PR on infra repo
+      - name: Create pull request
```

Comment:
ok for the moment, next step should be useful open PR iif file is changed

Reply:
there is [this condition ]( https://github.com/pagopa/pagopa-debt-position/pull/103/files/5370272406db6d5f4641d1e8600a6e830539b340#diff-214de57818e42135e36dd46f4416645f6bb84ab4afb3f789913b2c6f8ef798d6R8-R11)yet

Reply:
right , sorry